generators/app/templates/src/index.js

```diff
@@ -0,0 +1,51 @@
+export default {
+    // Multiple browsers support
+    isMultiBrowser: <%= isMultiBrowser %>,
+
+
+    // Required - must be implemented
+    // Browser control
+    async openBrowser (/* id, browserName, startPage */) {
+        throw new Error('Not implemented!');
+    },
+
+    async closeBrowser (/* id, pageInfo */) {
```

Comment:
So much to identify a window in `closeBrowser`... okay, if you think it's not so bad,,,